compiler/onecc-docker/debian/control

```diff
@@ -9,5 +9,5 @@ Homepage: https://github.com/Samsung/ONE
 Package: onecc-docker
 Architecture: amd64
 Multi-Arch: foreign
-Depends: ${misc:Depends}, ${shlibs:Depends}, python3.8
+Depends: ${misc:Depends}, ${shlibs:Depends}, python3
```

Comment:
:+1:

Reply:
Should the directory name be `compiler/onecc-docker/focal/control`?

Reply:
@dayo09 This file is located in `onecc-docker/debian/control`. It does not have OS version name in the directory path. :)

Reply:
Oh, my mistake. I read 'debian' as  'bionic'. :sweat: